ccm_web/server/src/app.module.ts

```diff
@@ -1,10 +1,14 @@
 import { Module } from '@nestjs/common'
-import { ConfigModule } from '@nestjs/config'
+import { ConfigModule, ConfigService } from '@nestjs/config'
+import { SequelizeModule } from '@nestjs/sequelize'
 
 import { APIModule } from './api/api.module'
 import { LTIModule } from './lti/lti.module'
 
 import { validateConfig } from './config'
+import { UserModule } from './user/user.module'
+import { User } from './user/user.model'
+import { UserService } from './user/user.service'
```

Comment:
I
```suggestion
import { APIModule } from './api/api.module'
import { LTIModule } from './lti/lti.module'
import { User } from './user/user.model'
import { UserModule } from './user/user.module'
import { UserService } from './user/user.service'

import { validateConfig } from './config'
```
I was thinking something like this. The idea is that some modules are Nest modules, and some are custom defined modules, and separating them might be beneficial. This isn't a super big deal.

Reply:
I think I am doing what you asked me to do. I will resolve it.